jobs/config.json

```diff
@@ -8074,6 +8074,28 @@
       "UNKNOWN"
     ]
   },
+  "ci-kubernetes-e2e-kops-aws-kopeiovxlan": {
+    "args": [
+      "kubetest",
+      "--",
+      "--deployment=kops",
+      "--extract=ci/latest",
+      "--kops-kubernetes-version=ci/latest",
+      "--up",
+      "--test",
+      "--down",
+      "--kops-cluster=e2e-kops-aws-kopeiovxlan.test-aws.k8s.io",
+      "--kops-args=--networking=kopeio-vxlan",
+      "--kops-state=s3://k8s-kops-jenkins/",
+      "--test_args=--ginkgo.focus=ConfigMap --ginkgo.skip=\\[Slow\\]|\\[Serial\\]|\\[Disruptive\\]|\\[Flaky\\]|\\[Feature:.+\\]",
+      "--v"
+    ],
+    "scenario": "execute",
```

Comment:
I think you'll still want to use `kubernetes_e2e` scenario here, unless we don't care anything in https://github.com/kubernetes/test-infra/blob/master/jobs/platform/kops_aws.env anymore? Looks like it will not dump logs properly.